### ISSUE TITLE
Generate person image from the segmetion one (COCO dataset)

### ISSUE THREAD

**Issue by QLinhHub** [junyanz/pytorch-CycleGAN-and-pix2pix#1288]
[](url)Hello, i have a problem when doing image translation from **SEGMENTATION** to **COLOR IMAGE** using Pix2Pix.
I have > 4000 images for training, my batch size is 16 and my learning rate is 0.0001.
After 200 epochs, the model generate well on training set, as bellow:
![image](https://user-images.githubusercontent.com/60795341/121905527-88a88c80-cd2a-11eb-9277-3530ce36d119.png)
But on test set, i have something like:
![image](https://user-images.githubusercontent.com/60795341/121905602-9f4ee380-cd2a-11eb-8414-3331f49fb3f6.png)
Is it the overfitting problem? If yes, what should i do in this case?
1. I think about add some regularizer layers but what value of lambda should i set?
2. Should i reduce my batch size to 1 or just use Instance Norm?
By the way, i'm working on **PERSON DEIDENTIFICATION** problem, is anyone working on this topic?
Every answer will be appreciate! Thanks in advance!

**Comment by QLinhHub** [comment]
Hello, can someone help me please !!!!

**Comment by synthetica3d** [comment]
Try using --norm instance because batch norm gave me endless headaches, and I found an issue with the batchnorm's behaviour after eval().
